src/main/cpp/include/vcf/genomicsdb_bcf_generator.h

```diff
@@ -34,7 +34,7 @@
 class GenomicsDBBCFGenerator {
  public:
   GenomicsDBBCFGenerator(const std::string& loader_config_file, const genomicsdb_pb::ExportConfiguration *query_config_pb,
-                         const char* chr, const int start, const int end,
+                         const char* chr, const long start, const long end,
```

Comment:
Did we miss this when the 64-bit support was implemented?

Reply:
Ah - hmm, possibly? Though I never remember what level of support htsjdk has for this...I made this change to sorta overload start/end (which used to only be contig start/end) to also play as tiledb column start/end (when chr is null or blank).